src/live_restore/live_restore_fs.c

```diff
@@ -919,6 +929,61 @@ __wti_live_restore_fs_fill_holes(WT_FILE_HANDLE *fh, WT_SESSION *wt_session)
     return (ret);
 }
 
+/*
+ * __wti_live_restore_cleanup_tombstones --
+ *     Remove all tombstone files from the database.
+ */
+int
+__wti_live_restore_cleanup_tombstones(WT_SESSION_IMPL *session)
+{
+    WT_DECL_RET;
+    WT_CONNECTION_IMPL *conn = S2C(session);
+    WTI_LIVE_RESTORE_FS *fs = (WTI_LIVE_RESTORE_FS *)conn->file_system;
+    WT_FILE_SYSTEM *os_fs = fs->os_file_system;
+    WT_SESSION *wt_session = (WT_SESSION *)session;
+
+    char **files;
+    char *logpath, *filepath;
+    uint32_t count = 0;
+    WT_DECL_ITEM(buf);
+
+    fs->finished = true;
+
+    /* Remove tombstones in the destination directory. */
+    WT_RET(os_fs->fs_directory_list(os_fs, wt_session, fs->destination.home, NULL, &files, &count));
+    for (uint32_t i = 0; i < count; i++) {
+        if (WT_SUFFIX_MATCH(files[i], WTI_LIVE_RESTORE_FS_TOMBSTONE_SUFFIX)) {
```

Comment:
Ideal world we have a
```c
if(is_directory(files[i]))
    WT_ASSERT(files[i] == log_path)
```
check here, but feel free to skip it if the implementation is non-trivial.

Alternately a recursive
```c
if(is_directory(files[i]))
    __wti_live_restore_cleanup_tombstones(files[i])
```
if we want to do away with the hardcoded log_path check